cmd/devtools/init.go

```diff
@@ -139,7 +167,35 @@ func recreateEnvFile(path string) {
 	if err != nil {
 		log.Fatalf("failed to write data to new file: %v", err)
 	}
-	fmt.Println("Local .env file created successfully.")
+	fmt.Println("\nLocal .env file created successfully.")
+}
+
+//go:embed config/remote.env.example
+var embeddedRemoteEnvironmentFile embed.FS
+
+func recreateRemoteEnvFile(instancDir string, path string) {
+	// Read the content from the embedded file
+	data, err := fs.ReadFile(embeddedRemoteEnvironmentFile, "config/remote.env.example")
+	if err != nil {
+		log.Fatalf("failed to read embedded file: %v", err)
+	}
+
+	// Specify the path for the new file
+	newPath := filepath.Join(path, ".env")
+
+	// Create a new file
+	newFile, err := os.Create(newPath)
+	if err != nil {
+		log.Fatalf("failed to create new file: %v", err)
```

Comment:
We should probably panic for things we consider fatal. There are a lot of these to add panics to I think.